addons/resource.language.en_gb/resources/strings.po

```diff
@@ -4831,7 +4831,31 @@ msgctxt "#2104"
 msgid "Check the log for more information."
 msgstr ""
 
-#empty strings from id 2105 to 2202
+#. Heading text for yes/no dialog to ask for add-on install to get his font file
```

Comment:
Can you reword this? It requires a deep understanding of how font language support works, might confuse translators.

Reply:
Have done a reword for here and on others.